staking/availability/measure.go

```diff
@@ -267,3 +267,27 @@ func UpdateMinimumCommissionFee(
 	}
 	return false, nil
 }
+
+// UpdateMaxCommissionFee makes sure the max-rate is at least higher than the rate + max-rate-change.
+func UpdateMaxCommissionFee(state *state.DB, addr common.Address, minRate numeric.Dec) (bool, error) {
```

Comment:
bool value returns from function, but never used. Maybe use it in tests?

Reply:
yeah good catch! i wanted to keep the same function signature as the `UpdateMinCommissionFee` but as you said if not used then we can remove it